yarn-project/circuits.js/src/structs/caller_context.ts

```diff
@@ -0,0 +1,80 @@
+import { AztecAddress } from '@aztec/foundation/aztec-address';
+import { type Fr } from '@aztec/foundation/fields';
+import { BufferReader, FieldReader, serializeToBuffer, serializeToFields } from '@aztec/foundation/serialize';
+import { type FieldsOf } from '@aztec/foundation/types';
+
+export class CallerContext {
+  constructor(
+    /**
+     * Address of the caller contract.
```

Comment:
```suggestion
     * Message sender of the caller contract.
```
This confused me, because I thought it was the address of the caller, but it's the caller of the caller instead :D